### PR TITLE
feat: Add ability to set extend_root_partition and remove_home_partition

### PR DESCRIPTION
**Pull request by pablomh**

### CODE REVIEW

**Comment by jhutar** [comment]
Hello! Should we have this for `satellite_remove_home_partition`, `satellite_extend_root_partition` and `capsule_...` counterparts? And when calling `remove-home-extend-root` role, just use it as values for plain `remove_home_partition` and `extend_root_partition`.

**Comment by jhutar** [comment]
We should probably add this to Capsule installation playbook for consistency as well.

File: playbooks/common/roles/remove-home-extend-root/tasks/main.yaml [hunk 1]
```diff
@@ -4,6 +4,7 @@
       - '!all'
       - hardware
 - include_tasks: remove_home_partition.yaml
+  when: 'remove_home_partition == True'
 - include_tasks: extend_root_partition.yaml
-  when: "extend_root_partition == 'true'"
+  when: 'extend_root_partition == True'
```
Comment (jhutar) [inline playbooks/common/roles/remove-home-extend-root/tasks/main.yaml:9]: Do we need explicit `...|bool` here?
```suggestion
  when: 'extend_root_partition|bool == True'
```
And if yes, what happens when `extend_root_partition` value is some string like `hello world`?

**Comment by jhutar** [comment]
If you do not think above is necessary, just ping me and we can merge this as it is nothing important.

**Comment by pablomh** [comment]
Hi Jan,

I've implemented the "casting" and decided to leave the rest as further improvement
because I (personally) prefer to have both Satellite and capsules configured as similar
as possible. I hope this approach works for your usecase as well.

**Pull request merged**